content-types/bar.yml

```diff
@@ -0,0 +1,31 @@
+name: Bar
```

Comment:
when testing using agent, we get content-types from punchcard content-types. With just one CT, I will not be able to test reference